### DISCUSSION TITLE
Oozie el to javascript el

### DISCUSSION THREAD

**Discussion by martini612** [Netflix/conductor#3434] [Q&A]
We are changing from Oozie engine to Conductor engine this time. How do you change Oozie EL to Javascript EL or Value-param?

**Comment by manan164** [comment]
Hi @martini612 , Can you please share couple of examples? We can convert it to Java Script EL. 
If you want to chat real time. Please join [community](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg)

**Reply by martini612** [reply]
Thank you for your response. I'll do some tests and leave the results in the comments.

Examples of conditional statements are as follows.

${fs:exists(concat(wf:conf('nameNode'), wf:conf(concat(wf:conf('ndap_user_name'),'_hdfs_user_data_path'))))}
${fs:exists(concat(wf:conf('nameNode'),wf:conf('hdfs_user_repo')))}
